libfly/binding.cc

```diff
@@ -535,18 +534,19 @@ extern "C"
     switch (message.GetEvent())
     {
     case v8::kPromiseRejectWithNoHandler:
-      printf("no handler!\n");
+      printf("Unhandled promise rejection:\n");
       printf("is native error? %s\n", error->IsNativeError() ? "true" : "false");
       printf("%s\n", *v8::String::Utf8Value(v8::Isolate::GetCurrent(), error));
       break;
-    case v8::kPromiseRejectAfterResolved:
-      printf("promise reject after resolved\n");
-      break;
     case v8::kPromiseHandlerAddedAfterReject:
       printf("promise handler added after reject\n");
+      printf("%s\n", *v8::String::Utf8Value(v8::Isolate::GetCurrent(), error));
```

Comment:
I think this might segfault. I saw it works fine on macos (on travis), but bombs on linux. Tried it locally and it happened. I assume `error` can sometimes be an invalid pointer. Not sure.

Reply:
Yeah you're right, I added that for debugging and forgot to remove, good catch!